client/src/store/aurige.js

```diff
@@ -14,5 +19,19 @@ export default {
       const { candidats } = result
       commit(SHOW_AURIGE_RESULT, candidats)
     },
+    async [AURIGE_UPLOAD_CANDIDATS] ({ commit, dispatch, state }, file) {
+      const data = new FormData()
+      data.append('file', file)
+      try {
+        const result = await api.admin.uploadCandidatsJson(data)
```

Comment:
Juste avant de faire l'appel, j'aime bien indiqué dans le state qu'une requête est en cours de traitement (regarde tous les `is*ing` dans les différents modules du store (isFetching, notamment) pour pouvoir afficher n'importe où ensuite dans l'application un indicateur de chargement.